File: Leetcode/1464_Maximum_Product_of_Two_Elements_in_an_Array.cpp
class Solution {
public:
    int maxProduct(vector<int>& nums) {
        int num1_idx = max_element(nums.begin(), nums.end()) - nums.begin();
        int num1 = nums[num1_idx] - 1;
        nums[num1_idx] = 0;
        int num2 = *max_element(nums.begin(), nums.end()) - 1;
        return num1 * num2;
    }
};

File: Leetcode/1464_Maximum_Product_of_Two_Elements_in_an_Array.py
from ast import List


class Solution:
    def maxProduct(self, nums: List[int]) -> int:
        idx1 = nums.index(max(nums))
        val1 = nums[idx1] - 1
        nums.remove(val1 + 1)
        val2 = max(nums) - 1
        return val1 * val2